src/store/sdp/index.js

```diff
@@ -90,6 +90,12 @@ const actions = {
             throw res.DisplayMsg;
         }
     },
+
+    async updateAvailability({ commit, state }, status) {
```

Comment:
we can rename `status` to `availableCount` as it has values like -1, 1 representing count